src/main/java/io/takari/maven/builder/smart/ReactorBuildStats.java

```diff
@@ -39,22 +40,22 @@ class ReactorBuildStats {
 
   private ReactorBuildStats(Map<String, AtomicLong> serviceTimes,
       Map<String, AtomicLong> bottleneckTimes) {
-    this.serviceTimes = ImmutableMap.copyOf(serviceTimes);
-    this.bottleneckTimes = ImmutableMap.copyOf(bottleneckTimes);
+    this.serviceTimes = unmodifiableMap(serviceTimes);
```

Comment:
not copy? this is just a view

Reply:
Intentional, because it is closer to the original, see blog post.